### PR TITLE
Biomes invoices

### PR DESCRIPTION
**Pull request by WRRicht3r**
Added invoices for SORA/ Polka Biome

### CODE REVIEW

**Comment by Tieumsan** [comment]
Some of the dates in the file title don't correspond to the dates in the file. Please review and correct

**Comment by WRRicht3r** [comment]
@Tieumsan updated the titles and added the latest invoice from Band

**Comment by WRRicht3r** [comment]
Hello @Tieumsan @takemiyamakoto can this PR be reviewed and approved please? An invoice within the commit has been paid

**Review by takemiyamakoto — APPROVED** [review]

File: Band Protocol/June 2023_Sora Maintenance and Infra Invoice.pdf
Comment (Tieumsan) [inline Band Protocol/June 2023_Sora Maintenance and Infra Invoice.pdf:1]: the Band Protocol/SORA Implementation Invoice.pdf file in the folder does not have the same naming (no date)

File: Polka Biome/Polka Biome - Community grant request Jan23.pdf
Comment (Tieumsan) [inline Polka Biome/Polka Biome - Community grant request Jan23.pdf:1]: naming and date still inconsistent

Comment (Tieumsan) [reply Polka Biome/Polka Biome - Community grant request Jan23.pdf:1]: please recheck the others as well

**Review by Tieumsan — CHANGES_REQUESTED** [review]

**Review by Tieumsan — APPROVED** [review]

**Pull request merged**